### ISSUE TITLE
[BUG] Issue with retrieving Item data

### ISSUE THREAD

**Issue by ClaraSmyth** [TheDrone7/shieldbow#45]
**Describe the bug**
When the client is initialized with with storage enabled. There is a problem with retrieving items from the storage.

**To Reproduce**
```ts
await client.initialize({
  cache: true,
  // storage: false,
  storage: { enable: { dragon: true, api: false } },
  region: "euw",
  logger: { enable: true, level: "DEBUG" },
  ratelimiter: {
    strategy: "spread",
    throw: true,
    retry: { retries: 5, retryDelay: 5000 },
  },
  fetch: {
    champions: false,
    items: true,
    runes: true,
    summonerSpells: true,
  },
})

const player = await client.summoners.fetchBySummonerName("marblae");
const matchIds = await client.matches.fetchMatchListByPlayer(
  player.playerId
);

const match = await client.matches.fetch(matchIds[0]); // Error occurs here
console.log(match);
```

**Expected behavior**
When fetching a match it should return a match but instead throws an error because of an issue with the Item data being undefined.

```console
[2023-03-08T20:16:35.375Z] TRACE    :: Fetching all items
[2023-03-08T20:16:35.375Z] TRACE    :: Fetching items from storage
[2023-03-08T20:16:35.375Z] TRACE    :: (Local fetch) File path: /home/clara/Repos/test/test2/data/items/13.4/en_GB/items.json, exists: true.
/home/clara/Repos/test/test2/node_modules/shieldbow/dist/structures/dragon/Item.js:210
        this.availability = client.maps.filter((m) => data.maps[m.mapId.toString()]);
                                                               ^

TypeError: Cannot read properties of undefined (reading '1')
    at <anonymous> (/home/clara/Repos/test/test2/node_modules/shieldbow/src/structures/dragon/Item.ts:134:60)
    at Array.filter (<anonymous>)
    at Item (/home/clara/Repos/test/test2/node_modules/shieldbow/src/structures/dragon/Item.ts:134:37)
    at ItemManager.fetchAll (/home/clara/Repos/test/test2/node_modules/shieldbow/src/managers/ItemManager.ts:39:22)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at ItemManager.fetch (/home/clara/Repos/test/test2/node_modules/shieldbow/src/managers/ItemManager.ts:66:21)
    at Client.initialize (/home/clara/Repos/test/test2/node_modules/shieldbow/src/client.ts:527:41)
    at async file:///home/clara/Repos/test/test2/[eval1]:12:1

Node.js v19.7.0
```

**Desktop (please complete the following information):**
 - OS: EndeavourOS Linux
 - Node.JS version: v19.7.0
 - Library version: 2.0.0
- Typescript version: 4.9.3

**Comment by TheDrone7** [comment]
Not an issue with storage, seems like the structure of `items.json` has been updated, will check for changes and update accordingly.

**Comment by TheDrone7** [comment]
Fixed, published in v2.0.1

**Comment by TheDrone7** [comment]
Also added some tests to avoid breaking this in the future.
